libs/drupal_acces.js

```diff
@@ -0,0 +1,136 @@
+var request = require('superagent');
+var logger = require('../libs/logger')(module);
+
+var DRUPAL_PROFILE_API_URL = process.env.DRUPAL_PROFILE_API_URL;
+var DRUPAL_PROFILE_API_USER = process.env.DRUPAL_PROFILE_API_USER;
+var DRUPAL_PROFILE_API_PASSWORD = process.env.DRUPAL_PROFILE_API_PASSWORD;
+
+function _parse_fron_anchor_tag(uid_string) {
```

Comment:
i'm not sure this is the right thing to do (parsing html with a regex), why dont we have a json response?